pkg/fields/selector.go

```diff
@@ -235,3 +236,26 @@ func parseSelector(selector string, fn TransformFunc) (Selector, error) {
 func OneTermEqualSelector(k, v string) Selector {
 	return &hasTerm{field: k, value: v}
 }
+
+// SelectorHolder is a wrapper around Selector that allows for
```

Comment:
I was sure I'd already left a comment here, but I don't see it now. Maybe it's sitting unsent in one of my many open tabs.

Anyway, what do you think about moving these holders to pkg/api/unversioned, where we have other similar types?